opentelemetry/src/lib.rs

```diff
@@ -55,40 +35,109 @@
 //! # }
 //! ```
 //!
+//! See the [examples](https://github.com/open-telemetry/opentelemetry-rust/tree/main/examples) directory for different integration patterns.
+//!
 //! See the [`trace`] module docs for more information on creating and managing
 //! spans.
 //!
 //! [`Span`]: crate::trace::Span
 //!
-//! # Metrics
-//!
-//!
-//! The [`metrics`] module includes types for recording measurements about a
-//! service at runtime.
-//!
-//! ### Creating instruments and recording measurements
-//!
+//! # Getting Started with Metrics
+//!
+//! The [`metrics`] module provides types for recording measurements about a
+//! service at runtime. Below are the key steps to report measurements using
```

Comment:
"Service" is an overloaded term. We should remove "about a service" from the sentence here to just say: `The `[`metrics`]` module provides types for recording measurements about a service at runtime. `

Reply:
Agree. It is copied from otel official website. I'll address it in the future.